composer.json

```diff
@@ -19,18 +19,18 @@
     },
     "autoload": {
         "psr-4": {
-            "OpenTracing\\": "./src/OpenTracing/"
+            "OpenTracing\\": "src/"
         },
-        "files": ["./src/OpenTracing/Tags.php", "./src/OpenTracing/Formats.php"]
+        "files": ["src/Tags.php", "src/Formats.php"]
     },
     "autoload-dev": {
         "psr-4": {
-            "OpenTracing\\Tests\\": "./tests/OpenTracing"
+            "OpenTracing\\Tests\\": "tests/"
         }
     },
     "scripts": {
-        "test": "./vendor/bin/phpunit tests",
-        "lint": "./vendor/bin/phpcs --standard=ZEND --standard=PSR2 --ignore=*/vendor/* ./",
-        "fix-lint": "./vendor/bin/phpcbf --standard=ZEND --standard=PSR2 --ignore=*/vendor/* ./"
+        "test": "phpunit tests",
+        "lint": "phpcs --standard=ZEND --standard=PSR2 --ignore=*/vendor/* ./",
+        "fix-lint": "phpcbf --standard=ZEND --standard=PSR2 --ignore=*/vendor/* ./"
```

Comment:
Removing the prefix to the binaries installed with `composer` is an unrelated change, but good. See https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands:

>**Note**: Before executing scripts, Composer's bin-dir is temporarily pushed on top of the PATH environment variable so that binaries of dependencies are easily accessible. In this example no matter if the `phpunit` binary is actually in `vendor/bin/phpunit` or `bin/phpunit` it will be found and executed.

Reply:
@localheinz could you open a PR about this?

Reply:
@jcchavezs 

Absolutely!